docs/client/getting-started.mdx

```diff
@@ -244,52 +291,65 @@ We recommend storing the user's addresses and creating a function like [`getAddr
 Once you have a route, you can execute it in a single function call by passing in the route, the user addresses for at least the chains the route includes, and optional callback functions. This also registers the transaction for tracking.
 
 ```ts
-await skipClient.executeRoute({
-  route,
+await executeRoute({
+  route: routeResult,
   userAddresses,
-  // Executes after all of the operations triggered by a user's signature complete.
-  // For multi-tx routes that require multiple user signatures, this will be called once for each tx in sequence
-  onTransactionCompleted: async (chainID, txHash, status) => {
+  getCosmosSigner,
+  getEvmSigner,
+  getSvmSigner,
+  onTransactionCompleted: async (txInfo) => {
```

Comment:
now that this callback has been updated, we could use the same syntax to extract the object fields as the others to be consistent

`onTransactionCompleted: async ({ txHash, chainId, status}) => {`